ui/plugins/job_details/main.html

```diff
@@ -5,6 +5,8 @@
       <span ng-if="line.content_type != 'TalosResult'" ng-switch on="line.content_type">
         <a ng-switch-when="link" title="{{line.value}}"
            href="{{line.url}}" target="_blank">{{line.value}}</a>
+
+        <a title="{{line.value}}" href="http://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[job_details.length - 1].value | encodeURIComponent}}" ng-if="line.value.endsWith('raw.log')"><br>Filter results in wptview</a>
```

Comment:
This link successfully loaded in wptview for me. One can see the results after adding a filter